src/ronny.js

```diff
@@ -38,7 +38,7 @@ bot.on('message', function(data) {
               if (typeof message === 'function')
                 message = message();
               if (message.params) {
-                params = message.params;
+                for (var attrname in message.params) { params[attrname] = message.params[attrname];}
```

Comment:
this will not work for nested values, correct?
Will this suffice for our usecase?